src/main/webapp/WEB-INF/tags/shared/feedbackResponseComment.tag

```diff
@@ -20,7 +20,7 @@
 </c:choose>
 <li class="list-group-item list-group-item-warning${frc.extraClass}" id="responseCommentRow-${divId}">
     <div id="commentBar-${divId}">
-        <span class="text-muted">
+        <span class="text-muted"> 
```

Comment:
See if you can undo all the changes that involves addition of a whitespace and nothing else.